week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,11 @@
+fetch("https://xkcd.now.sh/?comic=latest")
+  .then((response) => response.json())
+
+  .then((data) => {
+    let imageTag = document.createElement("img");
```

Comment:
very small thing, a tradition is to name the response from a fetch inside the promise, res or response or result. data may sound a bit weird

Reply:
Change to response, thanks for advice.

Reply:
perfect !